setup.cfg

```diff
@@ -64,6 +64,7 @@ max-line-length = 88
 select = B,C,D,E,F,I,W,
 ignore = E203,E501,W503,D202,B014,D400,D401,DAR
 application-import-names = autonomy,packages,tests,scripts
+per-file-ignores = __init__.py:F401
```

Comment:
ignore unused imports in `__init__.py` files

Reply:
Please remove here and use `noqa: {code}` in the code